EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc

```diff
@@ -37,6 +37,16 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf):
     fedUnpackList_.push_back(1118);
     fedUnpackList_.push_back(1120);
     fedUnpackList_.push_back(1122);
+    // HBHE uTCA
```

Comment:
can we instead of this change to a loop from MINHCALuTCAFEDID to MAXHCALuTCAFEDID ?
that's from 1100 to 1199.
The only reason not to would be that you'd expect the FEDs not listed here already to come and go and produce corrupt data.